index.js

```diff
@@ -40,8 +40,11 @@ var isInWatchMode = function(){
   var argv = yargs(process.argv)
       .alias('w', 'watch')
       .argv;
+  var hasWebpackDevServer = Array.prototype.filter.call(process.argv, function (arg) {
+    return arg.indexOf('webpack-dev-server') !== -1;
+  }).length > 0;
```

Comment:
💭 There's gotta be a better way to do this. I tried looking through yargs, and I think [`command`](https://www.npmjs.com/package/yargs#commandcmd-desc-builder-handler) might work here